code/game/gamemodes/shadowling/shadowling_abilities.dm

```diff
@@ -0,0 +1,519 @@
+			A << "<font color=#311648><b>Destroy Object</b> will instantly destroy an adjacent object.</font>"
+			A << "<font color=#311648><b>Hypnosis</b> will instantly enthrall a target from range.</font>"
+			A << "<font color=#311648><b>Plane Shift</b> will allow you to phase through the void between worlds, making you invisible and incorporeal.</font>"
+			A << "<font color=#311648><b>Glacial Blast</b> will function is an extremely upgraded version of Flash Freeze.</font>"
+			A << "<font color=#311648><b>Vortex</b> will tear open a temporary hole in reality.</font>"
+			A.mind.current.verbs += /mob/living/simple_animal/ascendant_shadowling/proc/shadowling_annihilate
+			A.mind.current.verbs += /mob/living/simple_animal/ascendant_shadowling/proc/shadowling_destroyobject
+			A.mind.current.verbs += /mob/living/simple_animal/ascendant_shadowling/proc/shadowling_hypnosis
+			A.mind.current.verbs += /mob/living/simple_animal/ascendant_shadowling/proc/shadowling_phase
+			A.mind.current.verbs += /mob/living/simple_animal/ascendant_shadowling/proc/shadowling_glacial_blast
+			A.mind.current.verbs += /mob/living/simple_animal/ascendant_shadowling/proc/shadowling_vortex
+			sleep(50)
+			SSshuttle.emergency.request(null, 0.3)
+
+
+
+//Ascendant abilities
+/mob/living/simple_animal/ascendant_shadowling/proc/shadowling_annihilate(var/mob/boom) //Insta-gib, 30s cooldown
+	set category = "Ascendant"
+	set name = "Annihilate (30s)"
+	var/mob/living/simple_animal/ascendant_shadowling/SHA = usr
+
+	if(usr == boom)
+		return
+	if(SHA.phasing)
+		usr << "<span class='warning'>You are not in the same plane of existence. Unphase first.</span>"
+		return
+
+	usr.visible_message("<font color=#311648>You direct a lance of telekinetic energy at [boom].</font>", \
+						"<span class='danger'>[usr]'s eyes flare as they gesture at [boom]!</span>")
+
+	boom.visible_message("<span class='userdanger'>[boom] explodes!</span>")
+	boom.gib()
+
+	usr.verbs -= /mob/living/simple_animal/ascendant_shadowling/proc/shadowling_annihilate
+	sleep(300)
+	usr.verbs += /mob/living/simple_animal/ascendant_shadowling/proc/shadowling_annihilate
+	usr << "<font color=#311648><b>Annihilate</b> can now be used again!</font>"
+
+
+/mob/living/simple_animal/ascendant_shadowling/proc/shadowling_destroyobject(var/obj/target) //Destroys an object instantly, 5s cooldown
+	set category = "Ascendant"
+	set name = "Destroy Object (5s)"
+	var/mob/living/simple_animal/ascendant_shadowling/SHA = usr
+
+	if(!in_range(usr, target))
+		usr << "<span class='warning'>You need to be closer to \the [target] to destroy it.</span>"
+		return
+	if(SHA.phasing)
+		usr << "<span class='warning'>You are not in the same plane of existence. Unphase first.</span>"
+		return
+
+	usr.visible_message("<font color=#311648>You break apart \the [target].</font>", \
+						"<span class='danger'>[usr] gestures at \the [target], which explodes violently!</span>")
+
+	qdel(target)
+
```

Comment:
Should it be 'exploding violently' if it's just qdel'd?

And shouldn't there be a check to make sure you're blowing up an object in here? qdelling mobs is a bad.